hazelcast/include/hazelcast/client/internal/socket/BaseSocket.h

```diff
@@ -125,12 +125,14 @@ class BaseSocket : public hazelcast::client::socket
             add_invocation_to_map(connection, invocation, message);
 
             auto& datas = message->get_buffer();
-            std::vector<boost::asio::const_buffer> buffers;
-            buffers.reserve(datas.size());
+
+            struct outbox_entry_t outbox_entry;
```

Comment:
`struct` keyword is not needed here.

```suggestion
            outbox_entry_t outbox_entry;
```

Reply:
fixed